synthtool/gcp/templates/java_library/renovate.json

```
@@ -69,5 +69,8 @@
     }
   ],
   "semanticCommits": true,
-  "masterIssue": true
+  "masterIssue": true,
```

Let's switch this to the non-deprecated config while we're at it
```suggestion
  "dependencyDashboard": true,
```